frontend/src/assets/translations/nl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "about": "Over ons",
+  "volunteers": "Vrijwilligers",
+  "contactus": "Contacteer Ons",
+  "signup": "Aanmelden",
+  "login": "Inloggen",
+  "findyourvolunteer": "VIND JE EEN VRIJWILLIGER",
+  "volunteerswhocanhelp": "VRIJWILLIGERS WELKE HET KAN HELPEN",
+  "skill": "Vaardigheid",
+  "city": "Stad",
+  "language": "Taal",
+  "whoarewe": "WIE ZIJN WE?",
+  "aboutbehelp1": "BeHelp is een dienst die reisden en asielverzoekers verbindt met vrijwilligers die zich kunnen delen met hun ervaring en vaardigheden pro bono.",
+  "aboutbehelp2": "Bij BeHelp u kunt vrijwilligers vinden vanuit alle landen.",
+  "ourservices": "Onze diensten",
+  "ourservices2": "Behelp is een groep van vrijwilligers die de mogelijkheid, tijd en wensen hebben om aan nieuwkomers met hun dagelijkse problemen te helpen.",
+  "hyfstudents": "Project van de studenten van HYF",
+  "kidane": "BeHelp maakte me weer eens eens. Na het verlaten van Eritrea, had ik een heel harde dag en ben ik niet in staat om psychologische hulp te krijgen die ik niet kon betalen. Op deze platform kon ik een psychotherapeut vinden die mij hulp kan geven aan mensen zoals ik. Ze spraken Tigrinya! Mijn sessies hebben mij geholpen om nieuwe mogelijkheden te zien en een meer levende leven te leven, zelfs als het nog veel lastig is om aan de nieuwe landen te adapteren.",
+  "levi": "De drama van reisden die zich uit hun landen verlaten heeft altijd meegenomen, ook omdat mijn vader een reisende was een keer! Vooral omdat ik een frans leerling ben, geef ik talenklassen en ben ik een vertaler van arabisch, de taal die ik in mijn leven leer.",
```

Comment:
De drama van vluchtelingen  die hun landen verlaten heeft me altijd emotioneel getroffen, omdat mijn eigen vader was ooit een vluchtling geweest. Ik ben nu een leerkracht die frans les geeft en ben ook een interpreter in arabisch, mijn moedertaal.